pwndbg/typeinfo.py

```diff
@@ -36,22 +36,31 @@ def is_pointer(value):
 @pwndbg.events.start
 @pwndbg.events.stop
 def update():
+
+    def lookup_first_type(types):
```

Comment:
I'd prefer pushing this up to the global scope as `lookup_type` which just takes `*args` so we can comma-separate (rather than passing a tuple) and instead of raising a custom exception, raise the most recent exception `except:` -> `except e` and `raise Exception` -> `raise e`.